pkg/web_app/lib/src/account.dart

```diff
@@ -90,10 +90,14 @@ void _initWidgets() {
       .getElementById('-account-login')
       ?.onClick
       .listen((_) => authenticationProxy.trySignIn());
-  document
-      .getElementById('-account-logout')
-      ?.onClick
-      .listen((_) => authenticationProxy.signOut());
+  document.getElementById('-account-logout')?.onClick.listen((_) async {
+    await authenticationProxy.signOut();
+    // force-reload page if it was not reloaded after signing out
+    await Future.delayed(Duration(seconds: 3));
```

Comment:
Why 3 seconds?

1 should be more than enough, no?

Reply:
Why not just reload?

Reply:
I guess I don't trust that the `signOut` enough to complete with everything :) I'll remove it...